lib/cloud_controller/opi/stager_client.rb

```diff
@@ -0,0 +1,58 @@
+require 'httpclient'
+require 'uri'
+require 'cloud_controller/diego/staging_request'
+require 'cloud_controller/opi/helpers'
+
+module OPI
+  class StagerClient
+    def initialize(opi_url, config)
+      @client = HTTPClient.new(base_url: URI(opi_url))
+      @config = config
+    end
+
+    def stage(staging_guid, staging_details)
+      staging_request = to_request(staging_guid, staging_details)
```

Comment:
You should probably carry the same `logger.info` messages here that are in `BbsStagerClient#stage` https://github.com/cloudfoundry/cloud_controller_ng/blob/3919b14ea002ef62c7869a7900d6834cc222e118/lib/cloud_controller/diego/bbs_stager_client.rb#L15 — when staging fails, these two messages are useful.